api/test/logs/provider_test.cc

```diff
@@ -17,20 +19,26 @@ namespace nostd = opentelemetry::nostd;
 
 class TestProvider : public LoggerProvider
 {
-  nostd::shared_ptr<Logger> GetLogger(nostd::string_view /* logger_name */,
-                                      nostd::string_view /* options */,
-                                      nostd::string_view /* library_name */,
-                                      nostd::string_view /* library_version */,
-                                      nostd::string_view /* schema_url */) override
+  nostd::shared_ptr<Logger> GetLogger(
+      nostd::string_view /* logger_name */,
+      nostd::string_view /* options */,
```

Comment:
should we remove `options`? I don't think we need to deprecate it as Logs API is still experimental

Reply:
I'm not sure why `options` is added here.Maybe we can alose remove it from all Logger's APIs.

Reply:
Yes, I think it would be good to remove it from all Logger's API.

Reply:
`options` is removed.